docs/modules/ROOT/pages/blocking-safe-by-default.adoc

```diff
@@ -57,13 +57,7 @@ code can interact with the event loop on a certain path.
 
 === Execution Strategy
 
-An link:{source-root}/servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ExecutionStrategy.java[Execution Strategy]
-has two primary purposes:
-
-. Define which interaction paths require offloading.
-. Optionally specify an
-link:{source-root}/servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Executor.java[Executor]
-to use for offloading. In absence of a specified `Executor`, ServiceTalk will use a default `Executor`.
+The primary purpose of an link:{source-root}/servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ExecutionStrategy.java[Execution Strategy] is to define which interaction paths require offloading.
 
 At a general transport layer (no application level protocol), only two offload paths are available:
```

Comment:
Our `io.servicetalk.transport.api.ExecutionStrategy` does not define flags for sending/receiving data. Is this statement still valid? Should we reword it to target `hasOffloads()` and `isCloseOffloaded()` flags?

Reply:
I have adapted it to use HTTP as an example.